src/Components/LeftSideBar.tsx

```diff
@@ -1,11 +1,12 @@
 import React from "react";
```

Comment:
Please make the styling correct. The search button and location icon should be in the left and right corners. Also, take a look at the overall styling for the component.